pom.xml

```diff
@@ -79,7 +79,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <declared.ozone.version>${ozone.version}</declared.ozone.version>
 
     <!-- Apache Ratis version -->
-    <ratis.version>2.0.0</ratis.version>
+    <ratis.version>2.1.0-80f6e4b-SNAPSHOT</ratis.version>
```

Comment:
Thanks @bshashikant and @bharatviswa504 for the effect in RATIS-1326 and RATIS-1356.